### DISCUSSION TITLE
How about supporting math equations?

### DISCUSSION THREAD

**Discussion by tristone13th** [charmbracelet/glow#339] [Ideas]

**Comment by wakatara** [comment]
I was also coming to ask about this feature. Support for LaTeX `$$E = mc^2$$` type blocks which render correctly would be *amazing*. I know this matched with my notetaking for classes would go a long way for making glow something I could use seamlessly from neovim (even in a telekasten preview window) to shred through notes and references. 

I imagine the multiline support in the terminal might be tricky, but feels like it's a pretty nifty thing to try as well and I think it would make glow even more awesome.

**Comment by kottory** [comment]
I saw an interesting repo https://github.com/jbyuki/nabla.nvim which supports math rendering in a very tricky way. I wonder if it is possible for glow.

**Comment by tristone13th** [comment]
> I saw an interesting repo https://github.com/jbyuki/nabla.nvim which supports math rendering in a very tricky way. I wonder if it is possible for glow.

I second.

**Comment by jjiangweilan** [comment]
This is the reason I haven't switched to glow! please make it possible

**Reply by josheppinette** [reply]
What are you using now?

**Reply by jjiangweilan** [reply]
I don't read markdown math a lot these days. I usually work in Neovim and it has [render-markdown.nvim](https://github.com/MeanderingProgrammer/render-markdown.nvim) which has latex support. It’s not perfect, though
